### DISCUSSION TITLE
Connected accounts and switching between them

### DISCUSSION THREAD

**Discussion by StorytellerCZ** [meteor/meteor#11516] [Feature Requests]
Allow users to link accounts in one app and switch between them. Same as Google or Twitter.

Flow in an app:
Select in settings to link an app -> get a login screen where the user enters credentials for the other account -> on successful login add the `userId` into on both users into linked accounts field -> user can now call a method to switch to the newly added account and then back.

I imagine this would have to be extension of #11515 in some shape to allow for moving between two users.

**Comment by filipenevola** [comment]
My answer to this would be similar to the one [here](https://github.com/meteor/meteor/discussions/11515#discussioncomment-966694)

Does this need to be in the core?

It seems that any package could implement this feature or am I missing something?

We could also use this space to discuss new community packages? Maybe this could be a good place to idealize new community packages as well.
